new/LIT6374Confessions.xml

```diff
@@ -0,0 +1,85 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6374Confessions" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="am" xml:id="t1">ለክርስቲያን፡ ሁሉ፡ ተስፋ፡ ኑዛዜ፡ የሚሆን፡ ጥቂት፡ ቃል፡</title>
+                <title xml:lang="am" corresp="#t1" type="normalized">Lakrǝstiyān hulu tasfā nuzzāze yammihon ṭǝqit q̄al</title>
+                <title xml:lang="en" corresp="#t1">A short word for all Christians that will be a hope and a colsolation</title>
+                <title xml:lang="en" corresp="t2">Collection of Biblical Confessions from two monks from Dabra Bezan monastery </title>
```

Comment:
I would opt for the standard Dabra Bizan spelling in the title, the variant spelling can be recorded in the text or the manuscript record.

Reply:
That makes sense